src/js/vre/chapter31/components/IntroductionPage.jsx

```diff
@@ -2,28 +2,68 @@ import React from 'react';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
+import { focusElement } from '../../../common/utils/helpers';
 import SaveInProgressIntro, { introActions, introSelector } from '../../../common/schemaform/save-in-progress/SaveInProgressIntro';
 import FormTitle from '../../../common/schemaform/components/FormTitle';
 import OMBInfo from '../../../common/components/OMBInfo';
 
 class IntroductionPage extends React.Component {
   componentDidMount() {
+    focusElement('.va-nav-breadcrumbs-list');
   }
   goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
   render() {
     return (
       <div className="schemaform-intro">
-        <FormTitle title="Apply for VR&E"/>
+        <FormTitle title="Apply for vocational rehabilitation for service-disabled Veterans"/>
```

Comment:
@peggygannon should we include the word "benefits" in this line?
On line 29 we say: `Follow the steps below to apply for vocational rehabilitiation benefits.`

Reply:
@cehsu Copy will likely to change after Peggy's review and the next round of research.

Reply:
@cehsu @rianfowler Let's leave the copy as is for now.